modelator_py/apalache/cli.py

```diff
@@ -24,7 +24,7 @@ def pure(self):
             self._stdin is not None
         ), "The pure interface requires json input in stdin"
         json_dict = stdjson.loads(self._stdin.read())
-
+        
```

Comment:
Did you run the hooks? I think the formatter would not have allowed this whitespace.

Reply:
I added hooks only today (so I guess it didn't pick up this). WIll fix it.

Reply:
Ah you mean github didn't pick up? Did you add extra hooks? There are these hooks already https://github.com/informalsystems/modelator-py/blob/main/.pre-commit-config.yaml

Reply:
I was not precise: I did not have pre-commit installed locally. Once I did install it, it was only working for future commits (I had to run `pre-commit run --all-files` to check previous commits, which I did now)